src/ngraph/pass/opset0_downgrade.cpp

```diff
@@ -212,6 +213,37 @@ bool pass::Opset0Downgrade::run_on_node(shared_ptr<Node> node)
         modified = true;
         break;
     }
+    case OP_TYPEID::TopK:
+    {
+        const auto tmp = as_type_ptr<op::v1::TopK>(node);
+        const auto axis = tmp->get_axis();
+        const auto sort_type = tmp->get_sort_type();
+        const auto index_elem_type = tmp->get_index_element_type();
+
+        bool comnpute_max;
+        switch (tmp->get_mode())
+        {
+        case op::v1::TopK::Mode::MAX: comnpute_max = true; break;
+        case op::v1::TopK::Mode::MIN: comnpute_max = false; break;
+        default:
+            NGRAPH_CHECK(true,
+                         "Mode ",
+                         static_cast<int>(tmp->get_mode()),
+                         " is not supported during TopK:1 to TopK:0 conversion. Node: ",
+                         *node);
```

Comment:
This check will always pass.
Since TopK::Mode is an enum it should be impossible to have an invalid value here.

Reply:
Because `clang` compiler check if all enum are used in swtich as default I removed this assert.